astropy/io/votable/tests/converter_test.py

```diff
@@ -146,6 +146,41 @@ def test_float_mask_permissive():
     assert c.parse('null', config=config) == (c.null, True)
 
 
+def test_double_array():
+    config = {'verify': 'exception'}
+    field = tree.Field(None, name='c', datatype='double', arraysize='3',
+                       config=config)
+    data = (1.0, 2.0, 3.0)
+    c = converters.get_converter(field, config=config)
+    assert c.output(1.0, False) == '1'
+    assert c.output(1.0, [False, False]) == '1 1'
```

Comment:
Is this desirable?

Reply:
No.  Good point.  We really only want the broadcast for the case where a single mask needs to be broadcast over multiple values.

While I can't think of how this case could happen in the wild, it might be worth adding a check to do the broadcast only if `mask.size == 1`, and use the original `zip()` otherwise.

Reply:
Great idea, thanks!